contracts/src/v0.8/ccip/test/pools/GHO/GhoBaseTest.t.sol

```diff
@@ -51,18 +51,18 @@ abstract contract GhoBaseTest is BaseTest {
       emptyArray,
       router
     );
-    TransparentUpgradeableProxy tokenPoolProxy = new TransparentUpgradeableProxy(
```

Comment:
Small nit, but isn't the previous name clearer because it's a proxy for the tokenPool contract, same as how we have `tokenPoolImpl` ?

Reply:
you're right, thanks for the catch, didn't mean to commit this! [b56ab36](https://github.com/aave/ccip/pull/18/commits/b56ab36e0cbd958ec3c62eab4123d1bc5cc62854)